docs/user/dashboard/lens.asciidoc

```diff
@@ -48,389 +13,144 @@ To view the data summary for a field, click *i* next to the field.
-
-.. Drag and drop *clientip* to the *Vertical axis* field.
-
-. Change the *timestamp* interval.
-
-.. From the editor, click *timestamp* in the line chart layer.
-
-.. Select *Customize time interval*.
-
-.. Change the *Minimum interval* to `1 days`, then click *Close*.
-
-. Change the *Unique count of clientip* label and color.
-
-.. From the editor, click *Unique count of clientip*.
-
-.. In the *Display name* field, enter `Unique visitors` in the line chart layer.
-
-.. In the *Series color* field, enter *#CA8EAE*, then click *Close*.
-
-[discrete]
-[[configure-the-multiaxis-chart]]
-==== Configure the y-axes
-
-There is a significant difference between the *timestamp per day* and *Unique visitors* data, which makes the *Unique visitors* data difficult to read. To improve the readability, 
-display the *Unique visitors* data along a second y-axis, then change the formatting. When functions contain multiple formats, separate axes are created by default. 
-
-. From the editor, click *Unique visitors* in the line chart layer.
-
-. For *Axis side*, click *Right*, then click *Close*.
+image::images/lens_area_percentage.png[Lens Area percentage]
 
 [float]
-[[change-the-visualization-type]]
-==== Change the visualization type
+[[can-i-show-value-labels-for-my-chart]]
+===== How do I display value labels?
 
-. From the editor, click *Average of bytes* in the area chart layer. 
+A subset of *Lens* visualizations support value labels. Value labels are unsupported in date histogram.
 
-. From the *Value format* dropdown, select *Bytes (1024)*, then click *Close*.
+* *Bar* and *Horizontal Bar*
 +
 [role="screenshot"]
-image::images/lens_end_to_end_3_4.png[Multiaxis chart]
+image::images/lens_value_labels_xychart_toggle.png[Lens Bar chart value labels menu]
 
-[discrete]
-[[lens-legend-position]]
-==== Change the legend position
-
-The visualization is done, but the legend uses a lot of space. Change the legend position to the top of the chart.
-
-. From the *Legend* dropdown, select the top position.
+* *Pie*, *Donut*, and *Treemap*
 +
 [role="screenshot"]
-image::images/lens_end_to_end_3_5.png[legend position]
-
-. *Save* the panel, enter `Average Bytes vs. Unique Visitors` in the *Title* field, then add the panel to the dashboard.
-
-[discrete]
-[[percentage-stacked-area]]
-=== View the health of your website 
+image::images/lens_value_labels_partition_toggle.png[Lens Pie chart value labels menu]
 
-To detect unusual traffic, bad website links, and server errors, create a percentage stacked area chart that displays the associated response codes.
-
-. Open *Lens*.
-
-. From the *Available fields* list, drag and drop the data fields to the *Drop a field or click to add* fields in the editor.
-
-.. Drag and drop *Records* to the *Vertical axis* field.
-
-.. Drag and drop *@timestamp* to the *Horizontal axis* field.
+[float]
+[[how-are-ip-address-sorted]]
+===== How does Lens sort IP addresses?
```

Comment:
This question looks like the kind of question we would get internally, but I think the question our users have is much more basic: it's usually "why isn't my data sorted the way I want?" or "why is the sorting not working?". This is because each Lens chart type sorts differently, and we need to how the data table sorting _is not_ changing the ES results.

If you want to use IP addresses as an example of sorting, then feel free, but I think this answer needs to be expanded

@dej611

Reply:
Removing for 7.12. Let's address this for 7.13.